exercises/perfect-numbers/test/test_perfect_numbers.c

```diff
@@ -0,0 +1,87 @@
+#include "vendor/unity.h"
+#include "../src/perfect_numbers.h"
+
+void test_smallest_perfect_number_is_classified_correctly(void)
+{
+   TEST_ASSERT_TRUE(classify_number(6) == perfect_number);
```

Comment:
Small detail only, but I would make these of the form `TEST_ASSERT_EQUAL(perfect_number, classify_number(6));`